MineLeapMod/src/com/nautigsam/mineleapmod/GameRenderHandler.java

```diff
@@ -1,32 +1,26 @@
 package com.nautigsam.mineleapmod;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import com.nautigsam.mineleapmod.helpers.ModVersionHelper;
-import com.nautigsam.mineleapmod.inputevent.ControllerBinding;
+import com.nautigsam.mineleapmod.helpers.LogHelper;
+import com.nautigsam.mineleapmod.helpers.McObfuscationHelper;
+import com.nautigsam.mineleapmod.inputevent.ControllerInputEvent;
+import com.nautigsam.mineleapmod.lwjglVirtualInput.VirtualMouse;
+import com.nautigsam.mineleapmod.minecraftExtensions.MineLeapConfigMenu;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.Gui;
 import net.minecraft.client.gui.GuiControls;
 import net.minecraft.client.gui.GuiScreen;
-import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraftforge.fml.client.FMLClientHandler;
 import net.minecraftforge.fml.common.ObfuscationReflectionHelper;
-
 import org.lwjgl.input.Controllers;
 import org.lwjgl.input.Mouse;
 
-import com.nautigsam.mineleapmod.helpers.LogHelper;
-import com.nautigsam.mineleapmod.helpers.McGuiHelper;
-import com.nautigsam.mineleapmod.helpers.McObfuscationHelper;
-import com.nautigsam.mineleapmod.inputevent.ControllerInputEvent;
-import com.nautigsam.mineleapmod.lwjglVirtualInput.VirtualMouse;
-import com.nautigsam.mineleapmod.minecraftExtensions.JoypadConfigMenu;
-
+import java.util.ArrayList;
+import java.util.List;
 
 
 public class GameRenderHandler
 {
-	private static Minecraft mc = Minecraft.getMinecraft();
+	private static Minecraft mc = FMLClientHandler.instance().getClient();

```

Comment:
When I tested the mod in my official Minecraft client, it worked only with this change. This code is a bit old so it's possible that the old way to get MC instance doesn't work on newer versions.